src/main/java/teammates/logic/backdoor/BackDoorLogic.java

```diff
@@ -152,9 +152,9 @@ public String persistDataBundle(DataBundle dataBundle)
             response = injectRealIds(response);
         }
         frDb.createFeedbackResponses(responses.values());
-        
-        for(FeedbackSessionAttributes session : sessions.values()){
-            updateRespondants(session.feedbackSessionName, session.courseId);
+
+        for (FeedbackResponseAttributes response : responses.values()) {
+            updateRespondants(response.feedbackSessionName, response.courseId);
```

Comment:
So this seems to be the guilty change. @unyoungwax I don't seem to get the reason for this. Why is this needed?